index.bs

```diff
@@ -1923,6 +1923,34 @@ This section defines an initial set of extensions. These are recommended for imp
 interoperability.
 
 
+## FIDO AppId ## {#extension-appid}
+
+This authentication extension allows [RP]s who have previously registered an
+Authentication using the legacy FIDO JavaScript APIs to communicate with
+Authenticators.  Specifically, this extension allows [RP]s to specify an
+|appId| [[FIDO-APPID]] to overwrite the otherwise computed |rpId|.
+
+: Extension identifier
+:: `webauthn_appid`
```

Comment:
Done on renaming `webauthn_appid` to `fido_appid`

Done on correcting capitalization of `Authenticator` and `Client`